addons/mail/static/src/scss/followers.scss

```diff
@@ -110,6 +114,9 @@
                     @include o-position-absolute(6px, 18px);
                     cursor: pointer;
                 }
+                .o_inactive_partner_img{
```

Comment:
This should be in the o_partner section too

Reply:
Done!